src/libraries/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -651,8 +651,7 @@ private enum SequenceType
             MultiSegment = 0x00,
             Array = 0x1,
             MemoryManager = 0x2,
-            String = 0x3,
-            Empty = 0x4
+            String = 0x3
```

Comment:
```suggestion
            String = 0x3,
```

That's my preferred style too. I wasn't sure whether the runtime repo had a convention of omitting the trailing comma.